### PR TITLE
lesson 1

### PR DESCRIPTION
**Pull request by yuryver**

### CODE REVIEW

File: yury-ortuno/lesson-1/homework-1-contacs-manager/src/Contact.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ */
+public class Contact {
+    private String name;
+    private String phoneNumber;
+
+    void setName(String name) {
+        this.name = name;
+    }
+
+    String getPhoneNumber() {
```
Comment (eduatro) [inline yury-ortuno/lesson-1/homework-1-contacs-manager/src/Contact.java]: You should use an identifier access for this method (public or private)

File: yury-ortuno/lesson-1/homework-1-contacs-manager/src/Contact.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        return phoneNumber;
+    }
+
+    void setPhoneNumber(String phoneNumber) {
```
Comment (eduatro) [inline yury-ortuno/lesson-1/homework-1-contacs-manager/src/Contact.java]: You should use an identifier access for this method (public or private)

File: yury-ortuno/lesson-1/homework-1-contacs-manager/src/Contact.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
+/**
+ * Created by Yury Ortuno on 11/13/2018.
+ */
+public class Contact {
+    private String name;
+    private String phoneNumber;
+
+    void setName(String name) {
```
Comment (eduatro) [inline yury-ortuno/lesson-1/homework-1-contacs-manager/src/Contact.java]: You should use an identifier access for this method (public or private)

**Review by eduatro — CHANGES_REQUESTED** [review]

File: yury-ortuno/lesson-1/homework-1-contacs-manager/src/Contact.java (hunk 1 above)
Comment (yuryver) [reply yury-ortuno/lesson-1/homework-1-contacs-manager/src/Contact.java]: Fix Done

File: yury-ortuno/lesson-1/homework-1-contacs-manager/src/Contact.java (hunk 2 above)
Comment (yuryver) [reply yury-ortuno/lesson-1/homework-1-contacs-manager/src/Contact.java]: Fix done

File: yury-ortuno/lesson-1/homework-1-contacs-manager/src/Contact.java (hunk 3 above)
Comment (yuryver) [reply yury-ortuno/lesson-1/homework-1-contacs-manager/src/Contact.java]: Fix done

**Review by J4rv1sX — APPROVED** [review]

**Pull request merged**